### DISCUSSION TITLE
Univariate time series imputation

### DISCUSSION THREAD

**Discussion by Lakminikw** [IQSS/Amelia#27] [Q&A] [answered]
This is a very useful package. I would like to know whether is it possible to do multiple imputations for a univariate time series using this package? 
Any guidance/directions about the possibilities of doing that would be greatly appreciated. 

Also, I would like to know ideas about checking for the missing mechanism (MCAR, MAR, MNAR), particularly for a univariate time series.  

Thanks in advance!

**Comment by thegaryking** [comment, marked as answer]
Amelia works by connections with existing variables, and so if you use Amelia with a truly univariate time series, Amelia would fix the uncertainty estimates but the imputations would be relatively uninformative.  To improve, add some information. If the time series is smooth over time, you could add some linear, quadratic, spline or other functions as additional variables.  If an autoregressive process is reasonable, you could add lags (or leads) of the variable as additional variables.  Then Amelia would help more subject to these assumptions.
